ansible_collections/arista/cvp/molecule/cv_image_v3/test_cv_image_v3.yaml

```diff
@@ -0,0 +1,164 @@
+---
+- name: Test cv_image_v3
+  hosts: CloudVision
+  connection: local
+  gather_facts: no
+  vars:
+    # Make sure given image does not exist on CVP
+    image_name: "EOS-4.25.4M.swi"
+  tasks:
+
+  ### Run all the tasks sequentially ###
+
+    - name: "Create bundle {{inventory_hostname}}"
+      vars:
+        ansible_command_timeout: 1200
+        ansible_connect_timeout: 600
+      arista.cvp.cv_image_v3:
+        mode: bundle
+        action: add
+        bundle_name: Test_bundle
+        image_list:
+          - "{{image_name}}"
+      register: CREATE_BUNDLE_DATA
+    - name: "Gather CVP bundle information facts {{inventory_hostname}}"
+      vars:
+        ansible_command_timeout: 1200
+        ansible_connect_timeout: 600
+      arista.cvp.cv_image_v3:
+        mode: bundle
+        action: get
+      register: BUNDLE_DATA
+    - assert:
+        that:
+          - CREATE_BUNDLE_DATA.changed == true
+          - CREATE_BUNDLE_DATA.data.data == "Bundle : Test_bundle successfully created"
+          - CREATE_BUNDLE_DATA.failed == false
+          - BUNDLE_DATA.changed == false
+          - BUNDLE_DATA.data.bundles[0].name == "Test_bundle"
```

Comment:
looks like here we should make sure that the 0th index is our bundle as if we are testing this on a CVP cluster where there are multiple bundles there's no guarantee this will be always the first index

I managed to make this work doing the following:

```yaml
    - name: Fetch the tested bundle information
      set_fact:
        test_bundle: "{{ BUNDLE_DATA.data.bundles | json_query('[?name==`Test_bundle`]') | first }}"
    - assert:
        that:
          - CREATE_BUNDLE_DATA.changed == true
          - CREATE_BUNDLE_DATA.data.data == "Bundle : Test_bundle successfully created"
          - CREATE_BUNDLE_DATA.failed == false
          - BUNDLE_DATA.changed == false
          - test_bundle.name == "Test_bundle"
          - test_bundle.imageIds == ["{{image_name}}", "{{extension_name}}"]
```

we don't have a function in ansible side to fetch a specific image bundle and we are fetching all of them so we'd have to loop through them and figure out which one is ours, using `set_fact` and `json_query` I managed to do it like above. Note that I had to `pip install jmespath` to be able to use `json_query`

Reply:
Fixed!